backend/src/migrations/008_delete_reviews_with_removed_access_requests.ts

```diff
@@ -0,0 +1,25 @@
+import AccessRequestModel from '../models/AccessRequest.js'
+import ReviewModel from '../models/Review.js'
+import { removeAccessRequestReview } from '../services/review.js'
+
+export async function up() {
+  // get the access requests
+  const accessRequests = await AccessRequestModel.find({ deleted: false })
+  // get all reviews
+  const reviews = await ReviewModel.find({})
+  // Remove access request reviews where the access request is already deleted
+  for (const review of reviews) {
+    const reviewAccessRequestId = review.get('accessRequestId')
+    if (reviewAccessRequestId !== undefined) {
+      // accessRequestId field indicates this is an access request review
+      if (!accessRequests.some((accessRequest) => accessRequest.get('id') == reviewAccessRequestId)) {
```

Comment:
You can change this to

 `if (reviewAccessRequestId && !accessRequests.some((accessRequest) => accessRequest.get('id') == reviewAccessRequestId) { ... }`

Alternatively, you can keep them as separate conditions but throw some sort of error if you can't find `reviewAccessRequestId`